From cdc5ab8481b45d29e7c8e1d9033ec1a65fbd87a5 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Thu, 13 Mar 2014 17:38:13 +0800
Subject: [PATCH 297/304] ENGR00303362 Disable opengl preload in zygote

CTS testVroot will failed if preload OpengGL ES in zygote
Disable all GPU preload in zygote

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 imx6/etc/init.rc | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/imx6/etc/init.rc b/imx6/etc/init.rc
index 95d7447..62f5f76 100755
--- a/imx6/etc/init.rc
+++ b/imx6/etc/init.rc
@@ -390,6 +390,9 @@ on boot
     setprop ro.FSL_FLAC_PARSER 1
     setprop ro.FSL_MPG2_PARSER 1
 
+# don't preload OpenGL in Zygote, the Vivante drivers do not like it
+    ro.zygote.disable_gl_preload=true
+
 # ALSA mixer naming define
     setprop alsa.mixer.playback.master Playback
     setprop alsa.mixer.capture.master Capture
-- 
1.8.0

